src/libcore/alloc.rs

```diff
@@ -357,23 +360,28 @@ impl fmt::Display for AllocErr {
 /// The `CannotReallocInPlace` error is used when `grow_in_place` or
 /// `shrink_in_place` were unable to reuse the given memory block for
 /// a requested layout.
+// FIXME: should this be in libcore or liballoc?
```

Comment:
This is used in the `Alloc` trait and therefore needs to be in libcore.

Reply:
Indeed. Removed that comment.